jspm.browser.js

```diff
@@ -0,0 +1,8 @@
+SystemJS.config({
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["SystemJS" is not defined.](https://www.codacy.com/app/amenadiel/ig_markerfactory/file/2450782131/issues/source?bid=3207375&fileBranchId=3248248#l1)